internal/tfimport/tfimport_test.go

```diff
@@ -1015,3 +1015,37 @@ func TestImportersSanity(t *testing.T) {
 		}
 	}
 }
+
+func TestSimpleImporterTmplExtraField(t *testing.T) {
+	imp := &importer.SimpleImporter{
+		Fields: []string{"project", "role"},
+		Tmpl:   "{{.project}} {{.role}} {{.member}}",
```

Comment:
Add a note that the template is looking for `member`. And also below that it's missing `member`. It's not very obvious.

Reply:
I'll add a comment above the test that this is what it's testing.